src/graph.ts

```diff
@@ -575,8 +651,11 @@ export class FactoryGraph {
      * @see {@link TransferContainerNode}
      */
     createTransferContainer(items: Item[]): TransferContainerNode {
-        const name = "Transfer Container " + this.transferContainers.size
-        const container = new TransferContainerNode(name, items, this)
+        const container = new TransferContainerNode(
+            "Trans Container" + this.transferContainers.size,
```

Comment:
```suggestion
            `Trans Container${this.transferContainers.size}`,
```

Reply:
Fixed in https://github.com/lgfrbcsgo/du-factory-generator/commit/e3612727cf42e8f92e923180d0c72033bd0a1513